### ISSUE TITLE
Use more permanent url for paradigm page

### ISSUE THREAD

**Issue by Madoshakalaka** [UAlbertaALTLab/cree-intelligent-dictionary#143]
currently paradigm pages have url as: `/lemma/<lemma_id>` such as `/lemma/12311`

Lemma id is the id for the lemma stored in database. While they gaurantee one-to-one correspondence. They are highly subjective to change if we rebuild the database.

We should use  `/lemma/<wordform>` such as `/lemma/nipâw`

**Comment by Madoshakalaka** [comment]
Since both `pos` and `lc` fields could be unspecified in the database,

I propose the following scheme for middle part of the url:
```
if <lc> specified:
    use <lc> (such as vii, vai, nid)
else:
    if <pos> specified
        use <pos> (such as v, n)
    else
        use "etc"
```

**Comment by eddieantonio** [comment]
> If a url could match multiple lemmas, `pos` or `lc` or `analysis` or `id` should only be appended as url parameter only when necessary, each one of them gaurantees a increasingly stricter match with `id` that gaurantees a one-to-one match.
> 
> So the following are all legal urls:
> `/lemma/nipâw?pos=V`
> `/lemma/nipâw?lc=VAI`
> `/lemma/nipâw?analysis=nipâw+V+AI+Ind+Prs+3Sg`
> `/lemma/nipâw?id=302152`
> 
> All urls should fall back to search page if there are multiple matches or no lemma is matched.

I really like this suggestion! We'll add it to tasks, but prioritize it after the things we have to deal with now :/

**Comment by Madoshakalaka** [comment]
fixed by #295
